staff/joel-molero/js-arrays/find-index.js

```diff
@@ -0,0 +1,19 @@
+function findIndex(arr,elem){
+
+if (!arr.length) throw Error("array is empty");
+
+if (typeof array[index] !== 'undefined') 
+
+
+
+
+
+
+    for(x=0;x<arr.length;x++){
+        if(elem(arr[x])){
+            return x;
+        }
+        if()
```

Comment:
what is this "if" for? 
you also need to return -1 if no element satisfies the condition